Controller/Admin/TransactionTabData.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use Wirecard\Oxid\Model\Transaction;
+use Wirecard\Oxid\Controller\Admin\TransactionTab;
+
+use OxidEsales\Eshop\Application\Model\Order;
+use OxidEsales\Eshop\Application\Model\Payment;
+
+/**
+ * Controls the view for the data transaction tab.
+ */
+class TransactionTabData extends TransactionTab
+{
+    /**
+     * @inheritdoc
+     *
+     * @param Transaction $transaction
+     * @param Order       $order
+     * @param Payment     $payment
+     * @return array
+     */
+    public function getListData(Transaction $transaction, Order $order, Payment $payment): array
+    {
+        return [
+            [
+                'title' => $this->_i18n->translateString('maid'),
```

Comment:
I would prefer having a `translate` function in the base class that could be reused in its child classes for i18n purposes.